### ISSUE TITLE
FeedbackSubmission should not require Action field

### ISSUE THREAD

**Issue by caseykulm** [bazaarvoice/bv-android-sdk#25]
FeedbackSubmission does not support Action field, but requires it in the constructor currently. If Action.Preview is used then one FeedbackSubmission request is sent with Action.Preview. 

If Action.Submit is used then the typical chain of a FeedbackSubmission request with Action.Preview  being followed as is expected with other forms of submission that require validating a submission with the backend first. However, that is not the case with FeedbackSubmission, but is still going through as 2 successful FeedbackSubmission requests on the backend.

tl;dr Current way to avoid problem is to only use Action.Preview when making a FeedbackSubmission request. Next release however should avoid the problem all together.

**Comment by caseykulm** [comment]
Completed by https://github.com/bazaarvoice/bv-android-sdk/releases/tag/5.2.1
